### DISCUSSION TITLE
CustomRP immediatly closes whenever i open it.

### DISCUSSION THREAD

**Discussion by iAmAsherr** [maximmax42/Discord-CustomRP#222] [Q&A]
When i downloaded CustomRP and opened it, it didnt work (it wont open). Ive looked at another discussion about this, which did not work. Please help

**Comment by maximmax42** [comment]
Absolutely nothing shows up? Like at all?

**Reply by iAmAsherr** [reply]
Yep
I tried to check my locapappdata for it, but theres no localappdata/maximmax42 or any other thing that has to do with customrp :/

**Reply by maximmax42** [reply]
Weird. I'm not sure. Could be antivirus blocking it.

**Reply by iAmAsherr** [reply]
alr ima look into the antivirus

**Reply by iAmAsherr** [reply]
It isnt in any blocked programs on my antivirus. Probably because it doesnt install at all

**Reply by maximmax42** [reply]
It probably is, but not being allowed to run. Check %appdata%\CustomRP.

**Reply by iAmAsherr** [reply]
It also isnt in appdata

**Reply by maximmax42** [reply]
Oh. Then yeah, it didn't install. Run the installer (should be CustomRP.1.17.14.exe for latest version) again and say what do you see.

**Reply by iAmAsherr** [reply]
I runned it, but nothing opens

**Reply by maximmax42** [reply]
You are on Windows, right?
Maybe add the installer to antivirus exceptions, idk.

**Reply by iAmAsherr** [reply]
I am on windows, ima check

**Reply by iAmAsherr** [reply]
hmm tried to add it, but it still doesnt work

Maybe its just broken for my pc

**Reply by maximmax42** [reply]
Honestly dunno. It's weird.

**Reply by iAmAsherr** [reply]
Yeah, thanks for helpin tho

**Comment by Ineedhelpthisinsant101** [comment]
Having an issue with my customrp for discord. It was working fine for weeks but then it started saying that there was an error connecting and that the ID could be wrong even though I made sure that it was correct. So I reinstalled it but now I can't get past this, is there something I'm doing wrong? I'm not sure what's happening but there are no files that i can update. 
<img width="367" alt="Screenshot 2025-01-23 201134" src="https://github.com/user-attachments/assets/6bfc58fd-3d34-41c3-8235-b4554bd113d5" />
<img width="367" alt="Screenshot 2025-01-23 201204" src="https://github.com/user-attachments/assets/f58c0029-22ad-4946-8585-21dbf89df9a6" />
@maximmax42

**Reply by Ineedhelpthisinsant101** [reply]
i made an account just for this

**Reply by maximmax42** [reply]
> i made an account just for this

You could just join the discord server

As for the error, exit CustomRP before updating. It should be doing it automatically, but fails sometimes. If you can't find the app in the tray, kill it from task manager, CustomRP.exe

**Reply by Ineedhelpthisinsant101** [reply]
tysm I'll try it, hopefully it works. Should it solve the ID error as well? Also might be a dumb question but where do I find the discord link?

**Reply by Ineedhelpthisinsant101** [reply]
nevermind I just joined it
